purchase.adoc

```diff
@@ -1,174 +1,300 @@
-## Purchase API
+= Purchase API
 
-### Service Description
-The purchase API provides information about purchases made through iZettle.
+== Description
+
+====
+Provides information about purchases made through iZettle.
+====
+
+=== Base URL
 
-### URL
 https://purchase.izettle.com
 
-### Scopes
-The Purchase API implements the following scopes:
+=== OAuth scopes
 
 - `READ:PURCHASE`
 
-=== Get purchase history
-Will include cash register information if applicable. Purchases up to three years old are available to fetch.
+'''
+
+==== Fetch purchase history
+
+Returns a list of purchases for organization.
+Purchases up to three years old are available to fetch.
 
 `GET /purchases/v2`
 
-lastPurchaseHash (optional):: A value from "lastPurchaseHash" from the result of a previous history query, to continue listing purchases from the next record after the previous query.
-startDate (optional):: The start date (inclusive) for purchases to be retrieved from until today or endDate. By default startDate is resolved to three years back.
-endDate (optional):: The last date (exclusive) for purchases to be retrieved until.
-limit (optional):: The maximum number of records to return. Max value of limit is 1000.
-descending (optional):: When true, returns purchases with the highest timestamp first.  When false, returns purchases with the lowest timestamp first. Defaults to false if not specified.
+[%collapsible]
+====
+*Parameters*
+
+[cols="15%,10%,10%,10%,55%"]
+|===
+|Name|Type|In|Required/Optional|Description
+
+|lastPurchaseHash|string|query|optional|A value from "lastPurchaseHash" from the result of a previous history query, to continue listing purchases from the next record after the previous query.
+|startDate|string|query|optional|The start date (inclusive) for purchases to be retrieved from until today or endDate if provided. By default `startDate` is resolved to three years back.
+|endDate|string|query|optional|The last date (exclusive) for purchases to be retrieved until.
+|limit|string|query|optional|The maximum number of records to return. Max value of limit is 1000.
+|descending|string|query|optional|When true, returns purchases with the highest timestamp first.  When false, returns purchases with the lowest timestamp first. Defaults to false if not specified.
+|===
+
+
+*Responses*
+
+`Status: 200 OK` - Successful, response json is returned. +
```

Comment:
the response is returned.
If all responses are in JSON, readers know the response format.